Cargo.toml

```diff
@@ -7,12 +7,12 @@ license = "MIT"
 name = "dav1d"
 readme = "README.md"
 repository = "https://github.com/rust-av/dav1d-rs"
-version = "0.10.4"
+version = "0.11.0"
 rust-version = "1.78"
 
 [dependencies]
 bitflags = "2"
-dav1d-sys = { version = "0.8.2", path = "dav1d-sys" }
+dav1d-sys = { version = "0.8.3", path = "dav1d-sys" }
```

Comment:
Nothing changed in dav1d-sys so this shouldn't be necessary unless I'm missing something

Reply:
I tend to prefer to depend explicitly to the latest, but probably we can get away with leaving 0.8.2 if cargo agrees.

Reply:
Probably doesn't really matter either way, so whatever you prefer :) I generally try to keep dependencies as low as possible